CONTRIBUTING.rst

```diff
@@ -39,19 +39,26 @@ There is also a `pip pattern` for development mode:
 Testing
 =======
 
-Test Dependencies
------------------
+Dependencies
+------------
 
-Install serverspec dependencies:
+Tests will be skipped when the driver's binary is not present. `Serverspec`
+scenarios do require a working ruby, `Bunder`_, and `Rake`_.
+
+Install the test framework `Tox`_.
 
 .. code-block:: bash
 
-  $ bundle install
+  $ pip install tox
```

Comment:
Currently tox is installed via test-requirements.txt which from a developer standpoint is probably what we want to do.